rpc/src/test/scala/org/bitcoins/rpc/BitcoindInstanceTest.scala

```diff
@@ -18,8 +18,13 @@ class BitcoindInstanceTest extends AsyncFlatSpec with BeforeAndAfterAll {
   private implicit val actorSystem: ActorSystem = ActorSystem(
     "BitcoindInstanceTest")
 
-  private val sampleConf: Seq[String] =
-    Source.fromResource("sample-bitcoin.conf").mkString.split("\n")
+  private val source =
+    Source.fromURL(getClass.getResource("/sample-bitcoin.conf"))
+
+  private val sampleConf: Seq[String] = {
+    source.getLines.toSeq
```

Comment:
Is this tested locally? I believe I did something similar to this, and it didn't work

Reply:
It does pass for me locally. Check for yourself in the morning

Reply:
Works for me as well. I'm not sure what I did the other day, then

A small (not important, feel free to ignore) simplifIcation I found: 

```scala
private val source = Source.fromResource("sample-bitcoin.conf")
private val sampleConf = source.getLines
```